manifests/init.pp

```diff
@@ -197,6 +197,11 @@
 #
 # $email_conf::                 Email configuration file under /etc/foreman
 #
+# $tasks_service::              Name of the service for running the background Dynflow executor
+#                               type:String
```

Comment:
These types can be left out since you're using Puppet 4 types already which means this is redundant.